src/mo_frontend/typing.ml

```diff
@@ -847,13 +947,14 @@ and check_typ_bind_sorts env tbs =
   (* assert, don't error, since this should be a syntactic invariant of parsing *)
   List.iteri (fun i tb -> assert (i = 0 || (tb.T.sort = T.Type))) tbs;
 
-and check_typ_binds env typ_binds : T.con list * T.bind list * Scope.typ_env * Scope.con_env =
+and check_typ_binds env stable_scope typ_binds : T.con list * T.bind list * Scope.typ_env * Scope.con_env =
```

Comment:
I don't think `stable_scope` is used to enforce any check in the body. Is it redundant or the intended check unimplemented?

Reply:
Yes, a relict, I first thought that bounds would need to be stable too, but this is not necessary as the type parameter is already required stable if needed. Can be removed.